Wireframe/index.html

```diff
@@ -4,10 +4,43 @@
     <meta charset="UTF-8">
```

Comment:
Please add the doctype  tag: 
```suggestion
<!DOCTYPE html>
<html lang="en">
<head>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Wireframe</title>
    <link rel="stylesheet" href="style.css">
    <link rel="stylesheet" href="styles.css">
</head>
<body>
   <-- Body elements go here -->
</body>
</html>
```

Here is why it is important: https://www.geeksforgeeks.org/explain-the-importance-of-doctype-in-html/